http/http.go

```
@@ -88,8 +88,8 @@ func handleJSONError(w http.ResponseWriter, msg string, statusCode int, err erro
 
 func (s Server) authenticated(r *http.Request) (db.Author, bool, error) {
 	c, err := r.Cookie("authorName")
-	if err != nil {
-		return db.Author{}, false, err
+	if err != nil || c.Value == "" {
+		return db.Author{}, false, nil
```

Why do you ignore the error here?

`c.Value == ""` could be another if IMHO.